gips/data/aod/aod.py

```diff
@@ -206,7 +256,12 @@ def process_mean(cls, filenames, fout):
             totalvar[inds] = numpy.divide(totalvar[inds], counts[inds])
             imgout[1].Write(totalvar)
             t = datetime.datetime.now() - start
-            VerboseOut('%s: mean/var for %s files processed in %s' % (os.path.basename(fout), len(filenames), t))
+            utils.verbose_out(
+                '%s: mean/var for %s files processed in %s' %
```

Comment:
Optional:  Python 2 official docs says `"foo {} bar {}".format(...)` "should be preferred" over `"foo %s bar" % (...)"` in new code:  https://docs.python.org/2/library/stdtypes.html#str.format